ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/utils.ts

```diff
@@ -0,0 +1,15 @@
+import { VulnerabilityExceptionScope } from 'services/VulnerabilityExceptionService';
+
+export function getImageScope(scope: VulnerabilityExceptionScope): string {
```

Comment:
Nit: if you destructure `imageScope` in the function parameter, and `{ registry, remote, tag }` in the function body, it will clear out a lot of visual repetition in the logic.